tests/links008.bs

```diff
@@ -0,0 +1,29 @@
+<pre class=metadata>
+Title: Foo
+Group: test
+Shortname: foo
+Level: 1
+Status: LS
```

Comment:
FYI, I use the DREAM status for tests to avoid creating all the boilerplate that otherwise makes it very hard to see the important part of the output file. I'll merge this and just change/regen the tests, tho.

Reply:
...well, no, I was 100% wrong. This was literally the *only* test using the DREAM status; every single other test in the repo (that isn't specifically testing something about a particular Status) is LS. IGNORE ME.